### ISSUE TITLE
broken random seed input?

### ISSUE THREAD

**Issue by ExileFox** [hrishi-inamdar/OreRandomizerMod#3]
either i get a negative maximum value or zero. 

Mod seems broken because of this.

**Comment by hrishi-inamdar** [comment]
Hey the value in settings is supposed to be 0 to start. That's intentional. You can provide your own seed in that box if you like. 

Not sure about the negative value. Randomization works properly for me on the latest EA version even for seed 0. Let me know if I'm mistaken. 

Thanks.

**Comment by ExileFox** [comment]
No, I literally can't type anything other than zero into it, otherwise I
get it set to a very large negative number (think negative limit of a
signed integer or thereabouts) and because it turns negative, the box goes
red.
